README.md

```diff
@@ -1,87 +1,73 @@
-
-```kotlin
-val button: MaterialButton? = findView(R.id.button)
-val recyclerView: RecyclerView? by bind(R.id.recycler)
-```
-
-Also: `View.setBackgroundTint(color)` will wrap & tint a view's background drawable. 
-
-#### Prefs
-
-A bunch of generic-typed preference utility functions.
-
-```kotlin
-// editing a SharedPreferences object (automatically runs .apply())
-sharedPrefs.edit {
-    putInt("themeKey", THEME_DARK)
-}
+## Usage
 
-// setter/getter operator functions
-val theme: Int = sharedPrefs["themeKey"]
-sharedPrefs["themeKey"] = THEME_LIGHT
+#### Include code in your layout:
+
+```xml
+<Button
+        ohos:height="match_content"
+        ohos:width="match_content"
+        ohos:id="$+id:buttonimageutil"
+        ohos:below="$id:buttondimen"
+        ohos:right_of="$id:button"
+        ohos:top_margin="30vp"
+        ohos:top_padding="15vp"
+        ohos:bottom_padding="15vp"
+        ohos:left_padding="38vp"
+        ohos:right_padding="38vp"
+        ohos:left_margin="20vp"
+        ohos:text="Image Utils"
+        ohos:text_size="28vp"
+        ohos:text_color="#FFF8F4F4"
+        ohos:background_element="#FF03CACA"
+        />
 ```
 
-Property delegates can be used to greatly simplify SharedPreferences operations.
+## DimenUtils
 
-```kotlin
-// binding a single property
-val theme by sharedPrefs.get<Int>("themeKey", defaultValue = THEME_LIGHT)
+includes some unit conversion functions that use getDeviceCapability() or getDefaultDisplay() to find the display density:
 
-// uses the delegated property name ("themeKey") if a key is not provided
-val themeKey by sharedPrefs.get<Int>(defaultValue = THEME_LIGHT)
-```
+1. dpToPx(Float) : Int
 
-##### TypedPreference
+2. spToPx(Float) : Int
 
-TypedPreference objects can be used to create a set of typed "pointers" to reference throughout your project, instead of relying on a hardcoded preference key/string.
+3. pxToDp(Int) : Float
 
-```kotlin
-// constructs a TypedPreference<Int> with the key "PREF_THEME"
-val PREF_THEME by pref<Int>(defaultValue = THEME_LIGHT)
+4. pxToSp(Int) : Float
 
-// in another file...
-var theme by sharedPrefs.get(PREF_THEME)
 
-// alternatively, in an activity/fragment/etc (if using PreferenceManager.getDefaultSharedPreferences)
-var theme by prefs(PREF_THEME)
-```
+## Future Work
 
-#### DimenUtils
+1. Since there is no class in HarmonyOS similar to ClipDrawable and LayerDrawable, the SeekBarDrawable.java and SeekBarUtils.java are currently not implemented due to platform dependencies.
 
-Contains two `Context` extension functions: `getStatusBarHeight()` and `getNavigationBarHeight()`, that both work accordingly.
+2. In DimenUtil, since currently StatusBarHeight and NavigationBarHeight cannot be set in android resources as identifiers, getStatusBarHeight and getNavigationBarHeight method are currently not implemented due to platform dependencies.
```

Comment:
Why do we need to set StatusBarHeight and NavigationBarHeight in android resources ? Please change the statement.

Reply:
done